.github/workflows/testing.yml

```diff
@@ -359,15 +359,15 @@ jobs:
           docker rm taichi_build taichi_test -f
 
   build_and_test_windows:
-    name: Build and Test (Windows)
+    name: Build and Test Windows (${{ matrix.identifier }})
```

Comment:
In fact we can only keep the GPU one in the matrix since it will cover all backends we care about :D

Reply:
Agree